core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.ts

```diff
@@ -108,4 +114,27 @@ export class DotExperimentsConfigurationSchedulingAddComponent implements OnInit
             this.initialDate.setMinutes(30);
         }
     }
+
+    /**
+     * Initial end date should be at least 30 minutes after start date.
+     * @returns void
+     * @memberof DotExperimentsConfigurationSchedulingAddComponent
+     */
+    private setMinEndDate(): void {
+        if (this.form.value.startDate) {
+            this.minEndDate = new Date(this.form.value.startDate.getTime() + 1000 * 60 * 30);
+        } else {
+            this.minEndDate = null;
+        }
+
+        if (this.isStatDateMoreRecent()) {
+            this.form.patchValue({
+                endDate: null
+            });
+        }
+    }
+
+    private isStatDateMoreRecent(): boolean {
+        return this.form.value.startDate && this.form.value.startDate > this.form.value.endDate;
```

Comment:
Since that line of code is being repeated a few times, we can create a `getter`.

```suggestion
        return this.startDate && this.startDate > this.form.value.endDate;
```